### PR TITLE
ELECTRON-491 (Fixes a Javascript error when pop-in a meeting)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
Change method to use getFocusedWindow instead of fromWebContents [ELECTRON-491](https://perzoinc.atlassian.net/browse/ELECTRON-491)

## Approach
How does this change address the problem?
#### Problem with the code:
 - Calling a `fromWebContents` method with undefined `webContents` throwing an error
 
tng01_1

#### Fix:
 - Using `getFocusedWindow` for get the browser window access
 - Cleaned up spaces

## Spectron tests result
[Electron-491 — Spectron.pdf](https://github.com/symphonyoss/SymphonyElectron/files/2004714/Electron-491.Spectron.pdf)

## Unit tests result
[Electron-491 — Unit Tests.pdf](https://github.com/symphonyoss/SymphonyElectron/files/2004713/Electron-491.Unit.Tests.pdf)

## Open Questions if any and Todos
- [X] Unit-Tests
- [X] Documentation
- [X] Automation-Tests
When solved, check the box and explain the answer.

### CODE REVIEW

**Comment by VishwasShashidhar** [comment]
@VikasShashidhar Please review this.

File: js/windowMgr.js [hunk 1]
```diff
@@ -496,73 +496,76 @@ function doCreateMainWindow(initialUrl, initialBounds) {
...
+
                         log.send(logLevels.INFO, 'permission is -> ' + userPermission);
-                        
+
                         if (!userPermission) {
                             let fullMessage = `Your administrator has disabled ${message}. Please contact your admin for help.`;
-                            electron.dialog.showMessageBox(BrowserWindow.fromWebContents(webContents), {type: 'error', title: 'Permission Denied!', message: fullMessage});
+                            const browserWindow = BrowserWindow.getFocusedWindow();
+                            if (browserWindow && !browserWindow.isDestroyed()) {
+                                electron.dialog.showMessageBox(browserWindow, {type: 'error', title: 'Permission Denied!', message: fullMessage});
```
Comment (VikasShashidhar) [inline js/windowMgr.js]: So, if `browserWindow` is not accessible/available, does that mean we don't show the error message at all?

Comment (KiranNiranjan) [reply js/windowMgr.js]: Yes that's correct, We don't show the error message if the `browserWindow` is not available

Comment (VikasShashidhar) [reply js/windowMgr.js]: Ok. But, how are we going to communicate to the user that that particular functionality is unavailable to him/her?

Comment (KiranNiranjan) [reply js/windowMgr.js]: @VikasShashidhar This only happens when the user starts the meeting and then changes the focus on to a different application, but there is a red banner shown for a few secs 
screen shot 2018-05-21 at 1 49 22 pm

The electron dialog will be shown to the user when he/she interacts with any of the actions buttons (In this case the `browserWindow` will always be defined)

**Pull request merged**